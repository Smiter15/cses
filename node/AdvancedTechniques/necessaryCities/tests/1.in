10 9
4 6
4 8
2 8
2 10
10 1
9 1
9 3
5 3
7 5
