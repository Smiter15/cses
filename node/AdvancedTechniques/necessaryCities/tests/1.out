8
4 8 2 10 5 3 9 1 
